From: PixelOS <pixelos@rom>
Date: Thu, 12 Feb 2026 00:00:00 +0000
Subject: [PATCH] Add HIDE_DEVELOPER_STATUS constant to Settings.Secure

Adds the constant for the hide_developer_status setting key.

---
 core/java/android/provider/Settings.java | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/java/android/provider/Settings.java b/core/java/android/provider/Settings.java
index 1234567..abcdefg 100644
--- a/core/java/android/provider/Settings.java
+++ b/core/java/android/provider/Settings.java
@@ -5000,6 +5000,10 @@ public final class Settings {
      */
     public static final class Secure extends NameValueTable {
         
+        /** @hide */
+        public static final String HIDE_DEVELOPER_STATUS = "hide_developer_status";
+        
         //... existing constants ...
         
         /**
-- 
2.34.1
